### PR TITLE
Modify metronome Scheduler for moving into OMR

### PR DESCRIPTION
**Pull request by jason-hall**
Signed-off-by: Jason Hall <jasonhal@ca.ibm.com>

### CODE REVIEW

**Comment by dmitripivkine** [comment]
Jenkins test sanity xLinux jdk8

**Comment by dmitripivkine** [comment]
Jenkins test sanity AIX jdk11

**Comment by dmitripivkine** [comment]
Jenkins compile win

**Comment by dmitripivkine** [comment]
@charliegracie Would you please provide instructions how hook data should be modified instead of be moved to delegate?

**Comment by jason-hall** [comment]
This should be ready for review

File: runtime/gc_realtime/Scheduler.cpp [hunk 1]
```diff
@@ -20,37 +20,25 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-#include "j9.h"
-#include "j9cfg.h"
-#include "j9protos.h"
-#include "j9consts.h"
-#include "modronopt.h"
-#include "ModronAssertions.h"
```
Comment (dmitripivkine) [inline runtime/gc_realtime/Scheduler.cpp]: I would not remove this include, code in this file contains MM Assertions

Comment (jason-hall) [reply runtime/gc_realtime/Scheduler.cpp]: Okay, updated

**Comment by dmitripivkine** [comment]
Jenkins compile win

**Comment by dmitripivkine** [comment]
Jenkins sanity xLinux jdk8

**Comment by dmitripivkine** [comment]
Jenkins test sanity xLinux jdk8

**Comment by jason-hall** [comment]
While we're looking at those class forward declarations in Scheduler.hpp, I don't think "class ILogger" even exists and forward declaring "class Scheduler" before class Scheduler seems pointless.

**Comment by dmitripivkine** [comment]
Jenkins compile win

**Pull request merged**